OpenGL/src/Renderer.h

```diff
@@ -14,7 +15,16 @@
 #include "AudioEngine.h"
 #include "Shader.h"
 
-
+class Line {
+public:
+   glm::vec2 start;
+   glm::vec2 end;
+   glm::vec4 color;
+   Line(const glm::vec2& s, const glm::vec2& e, const glm::vec4& c)
```

Comment:
You don't need the constructor, use aggregate initialization with `{ }`.

Reply:
Great point